products/omero/downloads/index.html

```diff
@@ -5,6 +5,7 @@
         <div class="callout large primary" id="bg-image-omero">
             <div class="row column text-center">
                 <h1>OMERO 5.3.2 Downloads</h1>
+                <a href="{{ site.baseurl }}/2017/05/16/omero-5-3-2.html" class="large btn-red button sites-button hide-for-small-only" style="text-shadow: none;">Read the Release Announcement</a>
```

Comment:
I like the idea of linking internally. In terms of workflow, this means that the PR bumping the version of a product will need to contain both the new release announcement as a post and an update of the product page to link to this new page.